spec/services/projects/set_attributes_service_spec.rb

```diff
@@ -311,5 +313,29 @@
         end
       end
     end
+
+    context "when skip_custom_field_validation is true" do
+      let(:contract_options) { { skip_custom_field_validation: true } }
+      let(:project) { create(:project) }
+
+      it "deactivates custom field validations" do
```

Comment:
The new specs in this file check that a method is called. It does not verify anything else. If the implementation of that method - or the order in which this and other methods are called - this spec will not catch it.